velox/docs/develop/scalar-functions.rst

```diff
@@ -32,23 +32,28 @@ must be a reference. Function arguments must be const references. The C++
 types of the arguments must match Velox types as specified in the following
 mapping:
 
-==========  ==============
-Velox Type  C++ Type
-==========  ==============
-BOOLEAN     bool
-TINYINT     int8_t
-SMALLINT    int16_t
-INTEGER     int32_t
-BIGINT      int64_t
-REAL        float
-DOUBLE      double
-VARCHAR     StringView
-VARBINARY   StringView
-TIMESTAMP   Timestamp
-ARRAY       ArrayValReader
-MAP         SlowMapVal
-ROW         RowReader
-==========  ==============
+==========  ==============================  =============================
+Velox Type  C++ Argument Type               C++ Result Type
+==========  ==============================  =============================
+BOOLEAN     bool                            bool
+TINYINT     int8_t                          int8_t
+SMALLINT    int16_t                         int16_t
+INTEGER     int32_t                         int32_t
+BIGINT      int64_t                         int64_t
+REAL        float                           float
+DOUBLE      double                          double
+VARCHAR     StringView                      out_type<Varchar>
```

Comment:
This is really helpful! I didn't realize the argument and result types could be different for the same Velox type.